### PR TITLE
[android] Increase kFIFOSize in AudioDestination

### PR DESCRIPTION
**Pull request by jasonzhangxx**
The value of kFIFOSize was estimated, so it should be ok for us to increase the size. The change can be reverted once we have a better solution.

b/376903291

### CODE REVIEW

**Comment by jasonzhangxx** [comment]
A simple pr to unblock testing. We can either not merge it, or merge then revert it in the future.

File: third_party/blink/renderer/platform/audio/audio_destination.cc
Comment (borongc) [inline third_party/blink/renderer/platform/audio/audio_destination.cc:69]: Leave a TODO if we want to merge this PR. This issue will be gone once we rebase to m126+.

As the current Chromium uses `kFIFOSize = 128 * 128` and it uses the max of `callback_buffer_size_ + render_quantum_frames` and `kFIFOSize` as fifo_length.

**Review by johnxwork — APPROVED** [review]

File: third_party/blink/renderer/platform/audio/audio_destination.cc
Comment (borongc) [inline third_party/blink/renderer/platform/audio/audio_destination.cc:69]: nit: `rebasle` should be `rebase`

**Review by borongc — APPROVED** [review]

**Pull request merged**
